src/Command/CheckAllSchemaTemplatesNameFieldsCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Didn't we originally also want to add a naming convention check for the file names as well? Especially since we have this special case that names should only contain dashes as opposed to the namespace+name of the template which should only contain underscores.
And didn't we also want to validate that the file name correlates with the namespace+name?

Reply:
for history, I missed that this is a public repo and therefore should only contain official conventions according to the official avro spec